src/components/Modal.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
```

Comment:
@trumanshuck thank you so, so much, for going the extra mile to make this an actual component 🙏.